main.go

```diff
@@ -81,6 +84,11 @@ func main() {
 	pflag.CommandLine.AddGoFlagSet(flag.CommandLine)
 	pflag.Parse()
 
+	if err := logsv1.ValidateAndApply(logOptions, nil); err != nil {
```

Comment:
It looks like the upstream change relies on the logOptions method:
```suggestion
	if err := logOptions.ValidateAndApply(); err != nil {
```

Reply:
After further review logsv1 is the correct way to go here.